tools/kochdocs.nim

```diff
@@ -116,7 +119,9 @@ doc/estp.rst
 doc/idetools.rst
 doc/docgen.rst
 doc/koch.rst
-doc/backends.rst
+doc/backends/backends.rst
```

Comment:
May be the `backend.rst` should be renamed to `index.rst` so that we get a nicer URL `doc/backends/index.html`.

Reply:
/cc @Araq

Reply:
Yeah, I'll do that right away

Reply:
No. I don't enjoy telling my tools ``X/index.rst`` vs ``Y/index.rst`` vs ``ZZ/index.rst``, it's stupid. Names are better when they mean something and are not artifacts of silly technical limitations.